classes/session/SessionManager.php

```diff
@@ -234,25 +237,13 @@ public function gc(int $lifetime): bool
      */
     public function updateSessionCookie($sessionId = false, int $expireTime = 0): bool
     {
-        $domain = ini_get('session.cookie_domain');
-        // Specific domains must contain at least one '.' (e.g. Chrome)
-        if (strpos($domain, '.') === false) {
```

Comment:
I think this validation isn't needed (it's possible to setup the domain with `localhost`)

Reply:
I'd suggest reviewing the issue associated with the code. This is back in Bugzilla days:

http://wayback.archive-it.org/7100/20180509142027/https://pkp.sfu.ca/bugzilla/show_bug.cgi?id=8843